packages/front-end/services/features.ts

```diff
@@ -676,7 +679,10 @@ export function getDefaultRuleValue({
   const hashAttribute = hashAttributes.includes("id")
     ? "id"
     : hashAttributes[0] || "id";
-
+  let defaultDataSource = settings?.defaultDataSource;
+  if (datasources && !defaultDataSource && datasources.length === 1) {
```

Comment:
I think @jdorn wanted to just default to picking the first one even if the length was > 1

Reply:
I'm fine with this for now. Less risky